policyengine_uk/variables/household/demographic/person.py

```diff
@@ -375,3 +375,40 @@ class is_older_child(Variable):
     def formula(person, period, parameters):
         age = person("age", period)
         return (age >= 14) & (age < 18)
+
+
+class is_parent(Variable):
```

Comment:
Make this a new file (eventually we will split all these variables into their own files)